### ISSUE TITLE
Otter export has a message when pdf generation fails

### ISSUE THREAD

**Issue by rohanjha123** [ucbds-infra/otter-grader#599]
**Is your feature request related to a problem? Please describe.**
Last semester, lots of data 100 student's pdf generations wouldn't work for homeworks. This could happen because they added an extra image, weird latex, anything like that. Then, when they submitted the zip to gradescope, their pdf generation broke (normally due to latex, written as a fandol or wkhtmltopdf error) and their written answers didn't get sent to the appropriate assignment. This meant that they didn't receive a grade for the written questions in the assignment, causing a lot of manual regrades. As students have very limited understanding of otter/latex and often don't check their zips before submitting (especially near deadlines), I think it's very hard for them to know when their pdf doesn't generate.

**Describe the solution you'd like**
If a solutions pdf is required for an assignment, it would be great if the otter export cell checks for generation/latex errors and lets the student know if the pdf cannot be generated.

**Describe alternatives you've considered**
For this semester, data 100 is making students manually download filtered pdf's of their assignment and submit it to a separate gradescope assignment. This is extra work as students need to make 2 submissions (one is the zip for the autograder and the second is the pdf for manual grading), but it forces the students to check their pdf generation.

**Additional context**
N/A

**Comment by chrispyles** [comment]
`grader.export` does currently issue a warning when PDFs are not generated, but this does not stop it from generating the zip file. https://github.com/ucbds-infra/otter-grader/blob/master/otter/check/notebook.py#L368

Can you give more details on what changes you'd like?

**Comment by rohanjha123** [comment]
It would be great if like a warning message popped up (and we could customize the warning message ourselves, like "the pdf failed to generate, you must try x steps or take screenshots if nothing works) and the student had to click ok on the warning before generating the zip

**Comment by chrispyles** [comment]
This will be included in v5.1.0. To add this acknowledgement to an assignment, set `export_cell: require_no_pdf_ack: true` in the assignment config. To customize the message, instead set

```yaml
export_cel:
	require_no_pdf_ack:
		message: <your message here>
```

**Comment by rohanjha123** [comment]
Thanks so much Chris! Also tagging @mihranmiroyan as an fyi
